target/linux/lantiq/image/netgear.mk

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright (C) 2017 LEDE & Edward O'Callaghan
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+include $(INCLUDE_DIR)/image.mk
+
+DEVICE_VARS += NETGEAR_BOARD_ID NETGEAR_REGION NETGEAR_HW_ID
+
+define Build/append-hdr-rootfs
+  mkimage -A mips -O linux -C lzma -T filesystem -a 0x00 -e 0x00 \
+          -n 'LEDE RootFS' \
+          -d $(IMAGE_ROOTFS) $@.tmp
+  cat $@.tmp >> $@
+  rm $@.tmp
+endef
+
+define Device/lantiqNetgear
+  KERNEL := kernel-bin | append-dtb | lzma | uImage lzma | pad-offset 4 0 | pad-to 2162624
+  IMAGES := sysupgrade.bin factory.img
+  IMAGE/default := append-kernel | append-hdr-rootfs | pad-rootfs | append-metadata
```

Comment:
Please add the  Device/lantiqNetgear code to the Device/DM200 define. For now we only have a single netgear board and it doesn't make much sense to split this code.

It might be even better to move the whole code to image/Makefile as long as we only have a single device using the code.